LICENSE

```diff
@@ -0,0 +1,21 @@
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+1. Redistributions of source code must retain the above copyright notice,
```

Comment:
Since the license is referring to copyright notice above I guess we should add one also to avoid ambiguity.